### PR TITLE
Show a clickable "." when there is a blank domain in the top domain lists

### LINKED ISSUE #1241: Empty Domain is not clickable in the Top Blocked Domains widget
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/AdminLTE/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/AdminLTE/issues)).

**How familiar are you with the the source code relevant to this issue?:**

1

---
**Expected behaviour:**

In the "Top Blocked Domains" box on the homepage an entry for queries to the empty domain (string "") should somehow be clickable and take me to `/admin/queries.php?domain=`

**Actual behaviour:**

The entry for the empty domain shows up in the list, but as there is no text in the field it's actually not clickable. If I inspect the DOM I see a `<a href="queries.php?domain="></a>` is added, but with no text there it's not clickable.

**Steps to reproduce:**

Reproducible anytime queries to the empty domain shows up in the Top Blocked Domains widget.

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

N/A

**Troubleshooting undertaken, and/or other relevant information:**

N/A

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Fixes #1241

### CODE REVIEW

**Comment by DL6ER** [comment]
This works,
Screenshot from 2020-05-07 07-56-28
however, it will still show the empty domains in the Query Log it links to:
Screenshot from 2020-05-07 07-58-06

File: scripts/pi-hole/js/index.js [hunk 1]
```diff
@@ -686,7 +686,8 @@ function updateTopLists() {
         }
 
         domain = escapeHtml(domain);
-        url = '<a href="queries.php?domain=' + domain + '">' + domain + "</a>";
+        urlText = domain === "" ? "." : domain;
```
Comment (XhmikosR) [inline scripts/pi-hole/js/index.js:689]: Maybe try this here:

```suggestion
        urlText = domain ? domain : ".";
```

File: scripts/pi-hole/js/index.js [hunk 2]
```diff
@@ -720,7 +721,8 @@ function updateTopLists() {
...
```
Comment (XhmikosR) [inline scripts/pi-hole/js/index.js:724]:
```suggestion
        urlText = domain ? domain : ".";
```

**Comment by XhmikosR** [comment]
If the above doesn't fix the issue @DL6ER described, just drop the `urlText` variable and use a ternary in the `url` itself.

**Comment by DL6ER** [comment]
The issue I described exists on another page and is due to missing code there, not a conflict with the proposed change (which works as expected).

@XhmikosR I wonder if your suggestion (replace `domain === ""` by `domain`) is equivalent. The former does only check for an empty string whereas the latter would also be triggered by boolean `false` or `null` etc. which may be undesired.

**Comment by XhmikosR** [comment]
@DL6ER well, that's why I said `maybe try with...` :)

**Pull request merged**